### ISSUE TITLE
Add links to docs in error messages

### ISSUE THREAD

**Issue by scudette** [Velocidex/velociraptor#2651]
Also in the config wizard

**Comment by weslambert** [comment]
Yes, this would be good to address for the interactive configuration wizard around the GUI especially, since users need to go back and modify the server configuration file to allow external access to it. Otherwise, users might not be aware that the bind address  is set to 127.0.0.1 by default.
